src/common/components/Settings.tsx

```diff
@@ -2700,6 +2700,18 @@ export function InnerSettings({
                         >
                             <MyCheckbox onBlur={onBlur} />
                         </FormItem>
+                        <FormItem
+                            style={{
+                                display: isDesktopApp && isWindows ? 'block' : 'none',
+                            }}
+                            name='enableMica'
+                            label={t('Enable mica (Experimental)')}
+                            caption={t(
+                                '**Win11 only. If the mica effect is enabled, it is essential to set the `Theme` to `Follow the system`, as it is currently not possible to manually switch between the light and dark themes of mica.'
+                            )}
+                        >
+                            <MyCheckbox onBlur={onBlur} />
```

Comment:
‌‌How about placing this 'form item' below the 'theme selector form item'? This way, it would be easier for users to associate it with the theme.

Reply:
It sounds great!

Reply:
Resolved.